packages/http/src/router/index.ts

```diff
@@ -63,24 +65,32 @@ export const router: IRouter<IHttpOperation, IHttpRequest, IHttpConfig> = {
 
     if (requestBaseUrl) {
       if (matches.every(match => match.serverMatch === null)) {
-        throw new Error(NO_SERVER_CONFIGURATION_PROVIDED_ERROR);
+        throw ProblemJsonError.fromTemplate(
+          NO_SERVER_CONFIGURATION_PROVIDED_ERROR,
+          `No server configuration has been provided, although ${requestBaseUrl} as base url`
+        );
       }
 
       if (matches.every(match => !!match.serverMatch && match.serverMatch === MatchType.NOMATCH)) {
-        throw new Error(NO_SERVER_MATCHED_ERROR);
+        throw ProblemJsonError.fromTemplate(
+          NO_SERVER_MATCHED_ERROR,
+          `The base url ${requestBaseUrl} hasn't been matched with any of the provided servers`
```

Comment:
Is it a base URL?

Reply:
Hmm, hard to say. Technically yes it's a base url, such as

`test.com/stuff`, but since we ignore the `basePath` or whatever it's called in OAS2, it's rather an `origin`